src/plugins/plot/MctPlot.vue

```diff
@@ -456,6 +456,8 @@ export default {
         },
         updateRealTime(clock) {
             this.isRealTime = clock !== undefined;
+            //re-request data if we switch from/to realtime
+            this.updateDisplayBounds(this.timeContext.bounds());
```

Comment:
I _think_ we may only need to do this if we switch from fixed time to realtime. Because if we're switching from realtime to fixed time, then the bounds wouldn't have changed?

Reply:
Done